README.md

```diff
@@ -22,20 +23,16 @@ GoIV is an android app made for Pokémon GO that allows users to seamlessly calc
 ![Alt text](https://i.imgur.com/xXr9zzK.jpg "Advanced Information in Results Popup")
 
 ## Download
-### Latest Version: 3.1.0 ([Changelog](CHANGELOG.md))
+[![Github All Releases](https://img.shields.io/github/downloads/farkam135/goiv/total.svg?maxAge=2592000)](https://github.com/farkam135/GoIV/releases/latest)
+[![GitHub release](https://img.shields.io/github/release/farkam135/goiv.svg?maxAge=2592000)](CHANGELOG.md)
```

Comment:
1. When I hover these links I don't see the alt-text, just the link.
2. I'd rather have labels before the images just to be sure—the first label should be "Latest version" or "Latest release", and the second should be "Changelog".